<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>576</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Hearing Measurement</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QGroupBox" name="idGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>barcode</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="iDlabel">
         <property name="text">
          <string>Participant ID:</string>
         </property>
         <property name="buddy">
          <cstring>barcodeLineEdit</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="barcodeLineEdit">
         <property name="inputMask">
          <string>0 0 0 0 0 0 0 0;_</string>
         </property>
         <property name="text">
          <string>       </string>
         </property>
         <property name="maxLength">
          <number>15</number>
         </property>
         <property name="cursorPosition">
          <number>15</number>
         </property>
         <property name="clearButtonEnabled">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>80</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="serialPortGroupBox">
      <property name="title">
       <string>serial ports</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="0">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="serialPortLabel">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Serial Port:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="serialPortComboBox"/>
         </item>
         <item>
          <widget class="QPushButton" name="serialPortConnectButton">
           <property name="text">
            <string>Connect</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="measureGroupBox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>measure</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="measurmentControlHhorizontalLayout">
         <item>
          <widget class="QPushButton" name="measureButton">
           <property name="text">
            <string>Measure</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="saveButton">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="closeButton">
           <property name="text">
            <string>Close</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="measurementTableHorizontalLayout">
         <item>
          <widget class="QTableWidget" name="leftTableWidget">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="autoScroll">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <property name="gridStyle">
            <enum>Qt::SolidLine</enum>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="cornerButtonEnabled">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>New Column</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>New Column</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>New Column</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>New Column</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string>Left</string>
            </property>
           </item>
           <item row="1" column="0">
            <property name="text">
             <string>1 kHz Test</string>
            </property>
           </item>
           <item row="1" column="1">
            <property name="text">
             <string>--</string>
            </property>
           </item>
           <item row="1" column="2">
            <property name="text">
             <string>500 Hz</string>
            </property>
           </item>
           <item row="1" column="3">
            <property name="text">
             <string>--</string>
            </property>
           </item>
           <item row="2" column="0">
            <property name="text">
             <string>1 kHz</string>
            </property>
           </item>
           <item row="2" column="1">
            <property name="text">
             <string>--</string>
            </property>
           </item>
           <item row="2" column="2">
            <property name="text">
             <string>2 kHz</string>
            </property>
           </item>
           <item row="2" column="3">
            <property name="text">
             <string>--</string>
            </property>
           </item>
           <item row="3" column="0">
            <property name="text">
             <string>3 kHz</string>
            </property>
           </item>
           <item row="3" column="1">
            <property name="text">
             <string>--</string>
            </property>
           </item>
           <item row="3" column="2">
            <property name="text">
             <string>4 kHz</string>
            </property>
           </item>
           <item row="3" column="3">
            <property name="text">
             <string>--</string>
            </property>
           </item>
           <item row="4" column="0">
            <property name="text">
             <string>6 kHz</string>
            </property>
           </item>
           <item row="4" column="1">
            <property name="text">
             <string>--</string>
            </property>
           </item>
           <item row="4" column="2">
            <property name="text">
             <string>8 kHz</string>
            </property>
           </item>
           <item row="4" column="3">
            <property name="text">
             <string>--</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="rightTableWidget">
           <property name="verticalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="horizontalScrollBarPolicy">
            <enum>Qt::ScrollBarAlwaysOff</enum>
           </property>
           <property name="autoScroll">
            <bool>false</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::NoSelection</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <property name="wordWrap">
            <bool>false</bool>
           </property>
           <property name="cornerButtonEnabled">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>New Row</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>New Column</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>New Column</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>New Column</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>New Column</string>
            </property>
           </column>
           <item row="0" column="0">
            <property name="text">
             <string>Right</string>
            </property>
           </item>
           <item row="1" column="0">
            <property name="text">
             <string>1 kHz Test</string>
            </property>
           </item>
           <item row="1" column="1">
            <property name="text">
             <string>--</string>
            </property>
           </item>
           <item row="1" column="2">
            <property name="text">
             <string>500 Hz</string>
            </property>
           </item>
           <item row="1" column="3">
            <property name="text">
             <string>--</string>
            </property>
           </item>
           <item row="2" column="0">
            <property name="text">
             <string>1 kHz</string>
            </property>
           </item>
           <item row="2" column="1">
            <property name="text">
             <string>--</string>
            </property>
           </item>
           <item row="2" column="2">
            <property name="text">
             <string>2 kHz</string>
            </property>
           </item>
           <item row="2" column="3">
            <property name="text">
             <string>--</string>
            </property>
           </item>
           <item row="3" column="0">
            <property name="text">
             <string>3 kHz</string>
            </property>
           </item>
           <item row="3" column="1">
            <property name="text">
             <string>--</string>
            </property>
           </item>
           <item row="3" column="2">
            <property name="text">
             <string>4 kHz</string>
            </property>
           </item>
           <item row="3" column="3">
            <property name="text">
             <string>--</string>
            </property>
           </item>
           <item row="4" column="0">
            <property name="text">
             <string>6 kHz</string>
            </property>
           </item>
           <item row="4" column="1">
            <property name="text">
             <string>--</string>
            </property>
           </item>
           <item row="4" column="2">
            <property name="text">
             <string>8 kHz</string>
            </property>
           </item>
           <item row="4" column="3">
            <property name="text">
             <string>--</string>
            </property>
           </item>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
